.gitignore

```diff
@@ -28,7 +28,10 @@ Thumbs.db
 
 # Build              #
 ######################
-dist/*
+dist/background.js
+dist/content.js
+dist/main.js
+dist/popup.js
```

Comment:
How about something like this instead?

```gitignore
dist/
!dist/extension/
```

Not 100% sure it'll work though

Reply:
It should work AFAIK